GiniVision/Classes/GiniVisionDocument.swift

```diff
@@ -12,27 +12,31 @@ import Foundation
  * Document processed by the _GiniVision_ library.
  */
 
-@objc public protocol GiniVisionDocument:class {
-    var type:GiniVisionDocumentType { get }
-    var data:Data { get }
-    var previewImage:UIImage? { get }
-    var isReviewable:Bool { get }
-    var isImported:Bool { get }
+@objc public protocol GiniVisionDocument: class {
+    var type: GiniVisionDocumentType { get }
+    var data: Data { get }
+    var previewImage: UIImage? { get }
+    var isReviewable: Bool { get }
+    var isImported: Bool { get }
     
     func checkType() throws
 }
 
 // MARK: GiniVisionDocumentType
 
-@objc public enum GiniVisionDocumentType:Int {
+@objc public enum GiniVisionDocumentType: Int {
     case pdf = 0
     case image = 1
+    case qrcode = 2
```

Comment:
Can't you omit these values (0, 1, 2). It's should be the default.

Reply:
I hate it but we cannot get rid of them because `objc` enums must declare an integer raw type. And this should be a `objc` enum since it is used in an `objc` protocol (`GiniVisionDocument`), which is needed in an optional `objc` method (in `GiniVisionDelegate`)